octodns/processor/awsacm.py

```diff
@@ -0,0 +1,46 @@
+#
+# Ignores AWS ACM validation CNAME records.
+#
+
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from logging import getLogger
+
+from .base import BaseProcessor
+
+
+class AwsAcmMangingProcessor(BaseProcessor):
+    log = getLogger('AwsAcmMangingProcessor')
+
+    def __init__(self, name):
+        '''
+        processors:
+          awsacm:
+            class: octodns.processor.acme.AwsAcmMangingProcessor
+
+        ...
+
+        zones:
+          something.com.:
+          ...
+          processors:
+            - awsacm
+          ...
+        '''
+        super(AwsAcmMangingProcessor, self).__init__(name)
```

Comment:
Looks like `__init__` is overridden just to add the doc string? If so it's probably fine to put it on the class (above `log` to avoid having to override. We haven't done the greatest job with documentation (consistency) so it should be fine to live there until if/when we go through and improve things.